src/app/layout.tsx

```diff
@@ -44,12 +44,12 @@ export default function RootLayout({ children }: RootLayoutProps) {
             }}
         >
             <html lang="en" className={archivo.className}>
-                <body className="overflow-x-hidden bg-grey">
-                    <div id="root" className="flex flex-col items-center">
-                        <Header />
+                <body className="overflow-x-hidden bg-grey text-white">
+                    <Header />
+                    <div className="mx-auto min-h-screen w-responsive pt-32 md:pt-40">
```

Comment:
Have you verified that the page layout looks the same? Because we need to have `flex flex-col items-center` otherwise the page will not be laid out properly (unless you have managed to forego this).

Reply:
Yea it looks same like before. I have refactored this in each page. Some pages only need `mx-auto` to centre.

Reply:
Alright, that's good.